metadata-ingestion/src/datahub/utilities/source_helpers.py

```diff
@@ -148,3 +155,44 @@ def auto_materialize_referenced_tags(
             entityUrn=urn,
             aspect=TagKeyClass(name=tag_urn.get_entity_id()[0]),
         ).as_workunit()
+
+
+def auto_browse_path_v2(
+    stream: Iterable[MetadataWorkUnit],
```

Comment:
it might be useful to have an `enabled` bool that makes the entire method a passthrough when disabled

Reply:
Hmm in case we make it a config option? What's the advantage over removing the helper entirely?

Reply:
Discussed offline, adding now

Reply:
Actually, I'd rather just not call it. We're getting to the point where we need to call helpers in a different way because python doesn't have a pipe syntax. It's not pretty to read `a(b(c(d(e(f(get_workunits())))))`. Also, we should standardize helper order because our helpers are not necessarily commutative.

I'm thinking of something like:
```
def workunit_modifiers(stream: Iterable[MetadataWorkUnit], auto_status: bool = True, auto_browse_path_v2: bool = True): ...
```

Or perhaps we could put it on the `Source` level:
```
class WorkUnitModifier(str, Enum):
    STALE_ENTITY_REMOVAL = "STALE_ENTITY_REMOVAL"
    REPORT_WORKUNITS = "REPORT_WORKUNITS"
    CREATE_STATUS = "CREATE_STATUS"
    CREATE_TAG_KEY = "CREATE_TAG_KEY"
    CREATE_BROWSE_PATHS_V2 = "CREATE_BROWSE_PATHS_V2"

@property
def workunit_modifiers(self) -> Collection[WorkUnitModifier]:
    return {WorkUnitModifier.REPORT_WORKUNITS, WorkUnitModifier.CREATE_STATUS, ...}

@abstractmethod
def get_workunits_internal(self) -> Iterable[WorkUnit]:
    pass

def get_workunits(self) -> Iterable[WorkUnit]:
    return apply_workunit_modifiers(self.get_workunits_internal(), self.workunit_modifiers)
```